Library/Entitlement.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+    public class Entitlement : RecurlyEntity
+    {
+        public string AccountCode { get; private set; }
+
+        public CustomerPermission CustomerPermission { get; private set; }
+
+        public List<GrantedBy> GrantedBys { get; private set; }
```

Comment:
This would need to change to 
        `public GrantedBy GrantedBy { get; private set; }`

Reply:
That would get rid of those Github Action errors further down